plugins/extractors/http/http_extractor.go

```diff
@@ -59,6 +61,7 @@ type QueryParam struct {
 
 var sampleConfig = heredoc.Doc(`
 	request:
+      route_pattern: /api/v1/endpoint 
 	  url: "https://example.com/api/v1/endpoint"
```

Comment:
```suggestion
	  route_pattern: /api/v1/endpoint 
	  url: "https://example.com/api/v1/endpoint"
```

(consistent whitespace)